spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -71,6 +72,10 @@
         create_request
         expect(response).to have_http_status(:created)
       end
+
+      it 'sends a notification email' do
```

Comment:
Actually this only tests that it enqueues a Job, whether it sends an email or not is not what you are testing here.
That would correspond to a rent mailer test.